### PR TITLE
Adding build script for binaryextensions UBI8.3

### PR DESCRIPTION
**Pull request by asellappen**
Test cases:-
=====================

1.  Default tag  v2.2.0

sh binaryextensions_ubi_8.3.sh

 [root@73299066db64 ~]# sh binaryextensions_ubi_8.3.sh
Cloning into 'binaryextensions'...
remote: Enumerating objects: 561, done.
remote: Counting objects: 100% (159/159), done.
remote: Compressing objects: 100% (37/37), done.
remote: Total 561 (delta 148), reused 122 (delta 122), pack-reused 402
Receiving objects: 100% (561/561), 399.25 KiB | 2.11 MiB/s, done.
Resolving deltas: 100% (407/407), done.
Note: switching to 'v2.2.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at d311c31 v2.2.0 - boundation, changelog, version bump
 found to checkout
v12.22.3
6.14.13
added 90 packages from 108 contributors and audited 88 packages in 1.998s

17 packages are looking for funding
  run `npm fund` for details

found 4 vulnerabilities (2 low, 2 high)
  run `npm audit fix` to fix them, or `npm audit` for details
Build successfully
npm WARN using --force I sure hope you know what you are doing.
updated 5 packages in 1.442s

17 packages are looking for funding
  run `npm fund` for details

fixed 4 of 4 vulnerabilities in 88 scanned packages
binaryextensions@2.2.0
├─┬ https://bevry.me/fund
│ └── assert-helpers@4.10.0, kava@3.2.0, projectz@1.10.0, taskgroup@5.5.0, eachr@3.3.0, extendr@3.5.0, typechecker@4.9.0, safefs@4.2.0, badges@1.4.0, caterpillar@3.3.0, caterpillar-filter@3.2.0, caterpillar-human@3.2.0, githubauthquerystring@1.1.0, safefs@5.1.0, taskgroup@6.1.0, safeps@8.0.0
└─┬ https://github.com/sponsors/isaacs
  └── glob@7.1.6

Ran fund & audit fix successfully

> binaryextensions@2.2.0 test /root/binaryextensions
> node ./source/test.js

extensions
extensions ➞ read the file
extensions ➞ read the file ✔
extensions ➞ require the file
extensions ➞ require the file ✔
extensions ➞ parse the file
extensions ➞ parse the file ✔
extensions ➞ data had no text extensions
extensions ➞ data had no text extensions ✔
extensions ➞ data had correct format
extensions ➞ data had correct format ✔
extensions ➞ data had duplicates removed
extensions ➞ data had duplicates removed ✔
extensions ➞ data was sorted
extensions ➞ data was sorted ✔
extensions ✔

7/7 tests ran successfully, everything passed
Build & Test completed successfully.
[root@73299066db64 ~]#

2. To run on tag v4.15.0 to capture failure 

sh binaryextensions_ubi_8.3.sh v4.15.0

[root@73299066db64 ~]# sh binaryextensions_ubi_8.3.sh v4.15.0
Cloning into 'binaryextensions'...
remote: Enumerating objects: 561, done.
remote: Counting objects: 100% (159/159), done.
remote: Compressing objects: 100% (37/37), done.
remote: Total 561 (delta 148), reused 122 (delta 122), pack-reused 402
Receiving objects: 100% (561/561), 399.25 KiB | 2.08 MiB/s, done.
Resolving deltas: 100% (407/407), done.
Note: switching to 'v4.15.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 5d5d3cd v4.15.0 - boundation, changelog, version bump
 found to checkout
v12.22.3
6.14.13
added 385 packages from 627 contributors and audited 385 packages in 9.604s

94 packages are looking for funding
  run `npm fund` for details

found 12 vulnerabilities (4 moderate, 7 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
Build successfully
npm WARN using --force I sure hope you know what you are doing.
+ typedoc@0.21.4
added 7 packages from 9 contributors, removed 12 packages and updated 7 packages in 5.389s

93 packages are looking for funding
  run `npm fund` for details

fixed 12 of 12 vulnerabilities in 385 scanned packages
binaryextensions@4.15.0
├─┬ https://bevry.me/fund
│ └── @bevry/update-contributors@1.17.0, assert-helpers@8.1.0, eslint-config-bevry@3.22.0, filedirname@2.3.0, kava@5.12.0, make-deno-edition@1.2.0, projectz@2.16.0, textextensions@5.11.0, valid-directory@3.4.0, valid-module@1.14.0, getcontributors@2.22.0, fellow@6.20.0, getrepos@5.16.0, githubauthreq@5.18.0, native-promise-pool@3.15.0, editions@4.2.0, errlop@3.17.0, @bevry/ansi@2.1.0, editions@6.0.0, errlop@4.0.0, version-range@1.0.0, version-compare@1.0.0, get-current-line@6.2.0, event-emitter-grouped@4.15.0, taskgroup@7.16.0, unbounded@3.12.0, eachr@4.5.0, extendr@5.15.0, editions@2.3.1, typechecker@4.11.0, errlop@2.2.0, typechecker@6.4.0, typechecker@7.16.0, get-cli-arg@6.7.0, normalify@2.20.0, badges@4.23.0, caterpillar@6.5.0, @bevry/ansi@1.4.0, rfc-log-levels@3.14.0, readdir-cluster@3.13.0
├─┬ https://opencollective.com/typescript-eslint
│ └── @typescript-eslint/eslint-plugin@4.6.0, @typescript-eslint/parser@4.6.0, @typescript-eslint/experimental-utils@4.6.0, @typescript-eslint/scope-manager@4.6.0, @typescript-eslint/types@4.6.0, @typescript-eslint/typescript-estree@4.6.0, @typescript-eslint/visitor-keys@4.6.0
├─┬ https://opencollective.com/eslint
│ └── eslint@7.12.1
├─┬ https://github.com/sponsors/mysticatea
│ └── regexpp@3.1.0, eslint-utils@2.1.0
├─┬ https://github.com/sponsors/sindresorhus
│ └── globby@11.0.1, globals@12.4.0, strip-json-comments@3.1.1, spdx-license-list@6.3.0
├─┬ https://github.com/sponsors/jonschlinkert
│ └── picomatch@2.2.2
├─┬ https://github.com/sponsors/feross
│ └── run-parallel@1.1.10, buffer@5.7.0, ieee754@1.2.1, safe-buffer@5.2.1
├─┬ https://www.patreon.com/feross
│ └── run-parallel@1.1.10, buffer@5.7.0, ieee754@1.2.1, safe-buffer@5.2.1
├─┬ https://feross.org/support
│ └── run-parallel@1.1.10, buffer@5.7.0, ieee754@1.2.1, safe-buffer@5.2.1
├─┬ https://github.com/sponsors/epoberezkin
│ └── ajv@6.12.6
├─┬ https://github.com/chalk/chalk?sponsor=1
│ └── chalk@4.1.0
├─┬ https://github.com/chalk/ansi-styles?sponsor=1
│ └── ansi-styles@4.3.0
├─┬ https://github.com/sponsors/isaacs
│ └── glob@7.1.6, rimraf@3.0.2, glob@7.1.7
└─┬ https://github.com/sponsors/ljharb
  └── deep-equal@2.0.4, es-abstract@1.18.0-next.1, es-get-iterator@1.1.0, is-date-object@1.0.2, is-regex@1.1.1, object-is@1.1.3, object.assign@4.1.1, regexp.prototype.flags@1.3.0, side-channel@1.0.3, which-collection@1.0.1, which-typed-array@1.1.2, es-to-primitive@1.2.1, has-symbols@1.0.1, is-callable@1.2.2, object-inspect@1.8.0, string.prototype.trimend@1.0.2, string.prototype.trimstart@1.0.2, is-symbol@1.0.3, es-abstract@1.17.7, is-map@2.0.1, is-set@2.0.1, is-string@1.0.5, is-boolean-object@1.0.1, is-number-object@1.0.4, is-weakmap@2.0.1, is-weakset@2.0.1, available-typed-arrays@1.0.2, is-typed-array@1.1.3

Ran fund & audit fix successfully

> binaryextensions@4.15.0 test /root/binaryextensions
> node ./edition-es5/test.js

5:Error: Cannot find module '/root/binaryextensions/edition-es5/test.js'
Test failed.
[root@73299066db64 ~]#

3. random next to default tag passing as argument 
sh binaryextensions_ubi_8.3.sh v2.3.0

[root@73299066db64 ~]# sh binaryextensions_ubi_8.3.sh v2.3.0
Cloning into 'binaryextensions'...
remote: Enumerating objects: 561, done.
remote: Counting objects: 100% (159/159), done.
remote: Compressing objects: 100% (37/37), done.
remote: Total 561 (delta 148), reused 122 (delta 122), pack-reused 402
Receiving objects: 100% (561/561), 399.25 KiB | 2.13 MiB/s, done.
Resolving deltas: 100% (407/407), done.
Note: switching to 'v2.3.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at ef9b2c7 v2.3.0 - boundation, changelog, version bump
 found to checkout
v12.22.3
6.14.13
added 55 packages from 98 contributors and audited 55 packages in 1.849s

32 packages are looking for funding
  run `npm fund` for details

found 2 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
Build successfully
npm WARN using --force I sure hope you know what you are doing.
npm WARN simplytyped@3.3.0 requires a peer of typescript@>=2.8.0 but none is installed. You must install peer dependencies yourself.

added 4 packages from 4 contributors, removed 1 package and updated 7 packages in 2.337s

36 packages are looking for funding
  run `npm fund` for details

fixed 2 of 2 vulnerabilities in 55 scanned packages
binaryextensions@2.3.0
└─┬ https://bevry.me/fund
  └── @bevry/update-contributors@1.6.0, assert-helpers@6.5.0, kava@5.1.0, projectz@2.2.0, textextensions@4.2.0, valid-directory@2.0.0, getcontributors@2.23.0, fellow@6.21.0, getrepos@5.17.0, githubauthreq@5.19.0, native-promise-pool@3.16.0, editions@6.2.0, errlop@4.2.0, version-range@1.2.0, version-compare@1.2.0, errlop@3.1.0, event-emitter-grouped@4.3.0, taskgroup@7.4.0, unbounded@3.1.0, eachr@4.5.0, extendr@5.3.0, editions@2.3.1, typechecker@4.11.0, errlop@2.2.0, typechecker@6.4.0, typechecker@7.2.0, badges@4.5.0, caterpillar@5.3.0, caterpillar-filter@6.3.0, caterpillar-human@6.3.0, fellow@6.8.0, get-cli-arg@5.2.0, get-current-line@5.3.0, rfc-log-levels@3.2.0, normalify@2.7.0, readdir-cluster@3.1.0

Ran fund & audit fix successfully

> binaryextensions@2.3.0 test /root/binaryextensions
> node ./source/test.js

extensions
extensions ➞ read the file
extensions ➞ read the file ✔
extensions ➞ require the file
extensions ➞ require the file ✔
extensions ➞ parse the file
extensions ➞ parse the file ✔
extensions ➞ data had no text extensions
extensions ➞ data had no text extensions ✔
extensions ➞ data had correct format
extensions ➞ data had correct format ✔
extensions ➞ data had duplicates removed
extensions ➞ data had duplicates removed ✔
extensions ➞ data was sorted
extensions ➞ data was sorted ✔
extensions ✔

7/7 tests ran successfully, everything passed
Build & Test completed successfully.

### CODE REVIEW

File: b/binaryextensions/binaryextensions_ubi_8.3.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+if [ -d "binaryextensions" ] ; then
+  rm -rf binaryextensions
+fi
+
+#Install Dependencies
+if [ ! -d ~/.nvm ]; then
+    sudo yum update -y
+    sudo yum groupinstall 'Development Tools' -y
+    sudo yum install -y openssl-devel.ppc64le curl git
+    sudo curl https://raw.githubusercontent.com/creationix/nvm/v0.37.2/install.sh | bash
```
Comment (gururajrkatti) [inline b/binaryextensions/binaryextensions_ubi_8.3.sh:41]: were we not able to simply use "dnf module install nodejs:12" for installing nodejs. Rest all LGTM

Comment (asellappen) [reply b/binaryextensions/binaryextensions_ubi_8.3.sh:41]: both should be fine , Earlier i have used dnf only and worked fine. But i have seen Siddhesh released at as above at  https://github.com/ppc64le/build-scripts/pull/969/files  in one of his release recently ,So i thought of following the one which got merged .

**Review by gururajrkatti — APPROVED** [review]
ok

File: b/binaryextensions/binaryextensions_ubi_8.3.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : binaryextensions
+# Version               : 2.2.0
+# Source repo           : https://github.com/bevry/binaryextensions
+# Tested on             : UBI 8.3
+# Script License        : Apache License, Version 2 or later
+# Passing Arguments     : Passing Arguments: 1.Version of package,
```
Comment (seth-priya) [inline b/binaryextensions/binaryextensions_ubi_8.3.sh:8]: Let's avoid changing the script header ..

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: b/binaryextensions/binaryextensions_ubi_8.3.sh (hunk 2 above)
Comment (asellappen) [reply b/binaryextensions/binaryextensions_ubi_8.3.sh]: corrected the headers,pls check

**Review by seth-priya — APPROVED** [review]

**Pull request merged**